content/projects/java-specific/streams/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+content_type: project
+flavours:
+- java
+prerequisites:
+  hard:
+  - projects/semitone-challenge/basic-algorithm
+  - topics/java-specific/streams
+ready: true
+submission_type: repo
+tags:
+- data-structures
+- array-manipulation
+title: Java Streams
+---
+
+Remember your semitone challenge project? well, for this project we are about to make the semitone project a bit cooler by using other functions to process the very same data structures.
+
+## Instructions:
+
+Pretty simple. You need to use at least three operators of streams to refactor your implementation of the semitone difference project. Make sure you test your logic, especially the paths that use streams.
```

Comment:
Consider rewording:

Eg:  Update your semitone project implementation so that it makes use of at least three stream operators. As always, make sure you test your code thoroughly.